osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/InvocationCounterManager.java

```diff
@@ -59,21 +59,32 @@ public InvocationCounterManager(
    */
   public void initializeInvocationCounter(
       final MessageMetadata messageMetadata, final DlmsDevice device) throws OsgpException {
+    /*
+     * When the invocation counter is out of sync, the device closes the session.
```

Comment:
Is this true for all devices? Is "some devices close the session" more correct?

Reply:
changed